CGAL_Core/include/CGAL/CORE/BigInt.h

```diff
@@ -24,86 +24,80 @@
 #ifndef _CORE_BIGINT_H_
 #define _CORE_BIGINT_H_
 
+#include <CGAL/boost_mp.h>
```

Comment:
boost mp is available only in boost 1.70. What is the plan, require 1.70 if Core is used?

Reply:
What do we require today? And is that because it is what is in Linux distributions?

Reply:
With Andreas we have decided to document that Boost 1.70 is required, starting with CGAL 5.6.